.github/scripts/update_generation_config.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
 set -e
 # This script should be run at the root of the repository.
-# This script is used to update googleapis commit to latest in generation
+# This script is used to update parameters in generation
```

Comment:
nit: maybe we can expand this explanation to mention which parameters are affected

Reply:
Done.